### ISSUE TITLE
Problem with identity theft

### ISSUE THREAD

**Issue by area912** [mltframework/shotcut#1331]
Hi, I was going to download your software and when I typed "shotcut" on google, it sort this : 
![image](https://user-images.githubusercontent.com/68190799/187534027-b6ef6ffe-2613-4b0c-aea3-9645868b2877.png)

The website is for Filmora and not Shotcut but on the "front door", it is marked "Shotcut".
Hope this situation is going to get better, I reported it but I'm not sure if it was sent.

Thanks for taking time developping Shotcut.
Sorry for my english, I'm french

A+

**Comment by ddennedy** [comment]
Thanks. I have reported these to Google Legal in the past as trademark violations. I have also complained on social media about these unethical businesses. After some back and forth to prove my legitimacy and trademark, they said they took action to block anyone except my company Meltytech from using the word "Shotcut" in their ad. I guess they only applied it in the country where I do business - USA. (Meltytech is the company I made to buy code signing certificates and setup advertising on our website.)
